Source/Tools/FEXLoader/LinuxSyscalls/SignalDelegator.cpp

```diff
@@ -150,6 +150,38 @@ namespace FEX::HLE {
     return SigInfoLayout::LAYOUT_KILL;
   }
 
+  void SignalDelegator::HandleSignal(int Signal, void *Info, void *UContext) {
+    // Let the host take first stab at handling the signal
+    auto Thread = GetTLSThread();
+    SignalHandler &Handler = HostHandlers[Signal];
```

Comment:
This shadows `Handler` below. Can it safely be moved down to the 2 lines of code instead? Otherwise, some minor renaming will prevent accidental mixups here.

Reply:
Moved lower and renamed the temporary in the loop.